recipes-multimedia/x264/x264_%.bbappend

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Typo in commit description - `distribnuted`.

Reply:
Will fix.

Reply:
Patch V2